mock/py/mockbuild/exception.py

```diff
@@ -11,17 +11,16 @@
 
 
 class Error(Exception):
+    resultcode = 1
+
     "base class for our errors."
-    def __init__(self, msg, status=None):
-        Exception.__init__(self)
-        self.msg = msg
-        self.resultcode = 1
-        if status is not None:
-            self.resultcode = status
+    def __init__(self, *args):
+        super().__init__(*args)
+        if len(args) > 1:
+            self.resultcode = args[1]
 
     def __str__(self):
-        return self.msg
-
+        return str(self.args[0])
```

Comment:
Hmmm, technically ... yes. But the readeability of the code is much worse. You have to scroll up in the code to read what is args[0] when you will be debugging the code.

Reply:
I agree it is hard to read (even though I'm not sure the old code was better, the fact Exception takes named argument is very weird).

The purpose of this change is that we need to keep `__init__()` without arguments (as Exceptions normally are) otherwise we can not pickle => unpickle (automatically done by the multiprocessing module).